src/Apps/Notifications/NotificationsApp.tsx

```diff
@@ -6,11 +6,11 @@ import { Media } from "Utils/Responsive"
 import { NotificationsContextProvider } from "Components/Notifications/useNotificationsContext"
 import { NotificationQueryRenderer } from "Components/Notifications/Notification"
 import { useRouter } from "found"
-import { GridColumns, Column, Flex } from "@artsy/palette"
+import { GridColumns, Column, Flex, FullBleed } from "@artsy/palette"
 import { DESKTOP_NAV_BAR_HEIGHT } from "Components/NavBar/constants"
 
 const DESKTOP_HEIGHT = `calc(100vh - ${DESKTOP_NAV_BAR_HEIGHT}px)`
-const MIN_LIST_WIDTH = 340
+const MIN_LIST_WIDTH = 350
```

Comment:
Increasing slightly so the the artworks in the list view can be displayed in one row.